rubygems/rubygems.org
# Remove twitter icon from footer

### Bug
https://github.com/rubygems/rubygems.org/issues/4600 

### Goal
Remove twitter icon from footer 

### How to test: 
1. Run `rails server` 
2. Open browser and update link to `[LOCAL HOST]/rails/mailers` 
3. Click any of the links listed to see the footer 

### Before: 
![image](https://github.com/jacklynhma/rubygems.org/assets/29336370/6261bf3e-041e-49a0-9f61-7b0c667f4e41)

### After: 
Computer view: 
<img width="782" alt="Screenshot 2024-05-08 at 4 00 50 PM" src="https://github.com/jacklynhma/rubygems.org/assets/29336370/acc16f08-a62d-421b-8620-cff87cb8a0f4">

Mobile View: 
<img width="519" alt="Screenshot 2024-05-08 at 4 00 46 PM" src="https://github.com/jacklynhma/rubygems.org/assets/29336370/e4a5977d-649b-46e0-b682-ae364ecbda01">
